src/components/Resume/Experience.js

```diff
@@ -46,8 +42,8 @@ Experience.propTypes = {
   font: PropTypes.string.isRequired,
 };
 
-const mapStateToProps = state => ({
-  experience: state.resume.experience,
+const mapStateToProps = (state) => ({
+  experience: migrateSchema(state.resume).work,
```

Comment:
This should not be here. There is no need to include the `migrateSchema` function. Please revert this line. The `migrateSchema` method will be called outside from here.

Reply:
Sorry, If my code caused you trouble.I will change it.